test/contracts/aevm_3/acm_dutch_auction.aes

```diff
@@ -0,0 +1,25 @@
+contract DutchAuction =
+
+  record state = { amount : int,
+                   height : int,
+                   dec    : int,
+                   sold   : bool }
+
+  private function require(b : bool, err : string) =
+    if(!b) abort(err)
+
+  public function init(price, decrease) : state =
+    require(price > 0 && decrease > 0, "bad args")
+    { amount = price,
+      height = Chain.block_height,
+      dec    = decrease,
+      sold   = false }
+
+  public stateful function bid() =
+    require( !(state.sold), "sold")
+    let price =
+      state.amount - (Chain.block_height - state.height) * state.dec
+    require( Contract.balance >= price, "no money")
+    Chain.spend(Contract.creator, price)
```

Comment:
I see this is correctly copy of /test/contracts/acm_dutch_auction.aes , adapted. Ok.